### ISSUE TITLE
Creating a media_profile from receiver capabilities information might require schema adjustments

### ISSUE THREAD

**Issue by cristian-recoseanu** [AMWA-TV/nmos-sink-metadata-processing#10]
Most properties mentioned in the [media_profile](https://github.com/AMWA-TV/nmos-sink-metadata-processing/blob/28ceea56dbb72ed6d9b5943d4a5322864b4ca8ca/APIs/schemas/media_profile.json) schema seem to have counterparts in the receiver capabilities register ([NMOS Capabilities - Register](https://specs.amwa.tv/nmos-parameter-registers/branches/main/capabilities/)).

The issue with media_type was already raised in another issue here:  AMWA-TV/nmos-sink-metadata-processing#9

The other two I am worried about are:
* components
* channels

# Components

In terms of components the [NMOS Capabilities - Register](https://specs.amwa.tv/nmos-parameter-registers/branches/main/capabilities/) counterpart is `urn:x-nmos:cap:format:color_sampling` and the suggestion is to be first retrieved from the SDP attribute ‘a=fmtp:’ format-specific parameter ‘sampling’.

Note this parameter is defined as a simple enum [Color (sub-)sampling](https://www.iana.org/assignments/media-type-sub-parameters/media-type-sub-parameters.xhtml#media-type-sub-parameters-15) so my inclination would be to modify the schema to allow for this value to be set in the `media_profile` in this fashion (using one of the values in the enum) and maybe even change the name to match the parameter in the [NMOS Capabilities - Register](https://specs.amwa.tv/nmos-parameter-registers/branches/main/capabilities/) (`color_sampling`)

# Channels

As far as I can tell the [NMOS Capabilities - Register](https://specs.amwa.tv/nmos-parameter-registers/branches/main/capabilities/) counterpart is `urn:x-nmos:cap:format:channel_count`. Now this is not compatible with the current schema requirement for the `channels` property in the `media_profile` so this is impossible to fill in as of right now. My suggestion would be to change this to a `channel_count` parameter which only requires the count.

**Comment by N-Nagorny** [comment]
It makes sense but slices possibilities a bit (a user may take channel symbols from the raw EDID and put them to `channels` along with the channel count). I think we need to determine how this slicing is critical for the activity user stories.

**Comment by cristian-recoseanu** [comment]
I would like some more guidance on those uses cases. If we identify that some of them can be retrieved from sources other than receiver capabilities those need to be highlighted and maybe for those we can make multiple variants of properties (e.g. one can set just the channel count and one more details about the channels). This would give us a better chance of interoperability.

**Comment by prince-chrismc** [comment]
These parameters, in part, are covered in the BCP

- `urn:x-nmos:cap:format:color_sampling` is noted [here](https://github.com/AMWA-TV/nmos-edid-to-receiver-caps-mapping/blob/v1.0-dev/docs/1.0.%20EDID%20to%20Receiver%20Caps%20mapping.md#base-edid)
- `urn:x-nmos:cap:format:channel_count` is described [here](https://github.com/AMWA-TV/nmos-edid-to-receiver-caps-mapping/blob/v1.0-dev/docs/1.0.%20EDID%20to%20Receiver%20Caps%20mapping.md#audio-receivers)

So it would make sense to include them in the API as well 🤔 

***

I would say the audio channel mapping is beyond our current scope but as Andy point out it's an important topic in the near future.
I think it's fair to summarize we _need_ user stories before we can tackle that topic.

Is it acceptable to leave it "undocumented" and thus unsupported?

**Comment by N-Nagorny** [comment]
Take a look at #16 please. I tried to tackle this there.
